dat/ships/pirate_admonisher.xml

```diff
@@ -16,7 +16,7 @@
  <trail_generator x="-20" y="5" h="-5" always_under="1">pirate</trail_generator>
  <trail_generator x="-20" y="-5" h="-5" always_under="1">pirate</trail_generator>
  <fabricator>Skull and Bones</fabricator>
- <description>The stock Admonisher is a powerful vessel, especially for its size. Through the usual Skull and Bones refit the Pirate Admonisher features improved manoeuvrability, more weapons space, and slightly improved defenses, at a cost of cargo capacity and armour regeneration rates.</description>
+ <description>The stock Admonisher is a powerful vessel, especially for its size. Through the usual Skull and Bones refit the Pirate Admonisher features improved manoeuvrability, more weapons space and slightly improved defences, at a cost of cargo capacity and armour regeneration rates.</description>
```

Comment:
Grepping the code, it looks like we use `defense` all over instead of `defence`. However, it does seem like we should be using `defence` all over. (this is not a change request, just an idle thought)

Reply:
*heh* I'll make a little note in case I find time :) - though I'm note sure I have a way to find them easily not having grep here on Windows... gah, I need to get a Linux box again!

Should this be in the code as well as in the text is the big question... I guess for consistancy...